// with this example we generate an error
// EmptyErrorImpl {message: 'no elements in sequence', name: 'EmptyError'}
// which is the error generated by the primeNumbers Observable at the end ot the execution

import { Subject } from 'rxjs';
import { first } from 'rxjs/operators';

const observableNotEmitting = new Subject<any>();
observableNotEmitting.pipe(first()).subscribe(console.log, console.log, () => console.log('done'));

observableNotEmitting.complete();
